modules/ossm-configuring-jaeger.adoc

```diff
@@ -31,101 +30,148 @@ The default Jaeger parameters specified in the `ServiceMeshControlPlane` are as
-* *Elasticsearch* - For production use. This is the default option for the `Production` deployment strategy.
-
 [NOTE]
 ====
-The default template strategy in the `ServiceMeshControlPlane` resource is `AllInOne`. For production, the only supported storage option is Elasticsearch, therefore you must configure the `ServiceMeshControlPlane` to request the `production-elasticsearch` template strategy when you deploy {ProductShortName} within a production environment.
+The default template in the `ServiceMeshControlPlane` resource is the `AllInOne` deployment strategy which uses in-memory storage. For production, the only supported storage option is Elasticsearch, therefore you must configure the `ServiceMeshControlPlane` to request the `production` template when you deploy {ProductShortName} within a production environment.
 ====
 
-Elasticsearch is a memory intensive application. The initial set of nodes created by the {product-title} installation may not be large enough to support the Elasticsearch cluster. Additional nodes must be added to the cluster if you want to run with the recommended amount (or more) memory. Each Elasticsearch node can operate with a lower memory setting though this is not recommended for production deployments.
 
-You should modify the default Elasticsearch configuration to match your use case. You can adjust both the CPU and memory limits for each component by modifying the resources block with valid memory and CPU values.
+[id="ossm-configuring-jaeger-elasticsearch_{context}"]
+== Configuring Elasticsearch
+
+The default Jaeger deployment strategy uses the `all-in-one` template so that the installation can be completed using minimal resources.  However, because the `all-in-one` template uses in-memory storage, it is only recommended for development, demo, or testing purposes and should NOT be used for production environments.
 
+If you are deploying {ProductShortName} and Jaeger in a production environment you must change the template to the `production` template, which uses Elasticsearch for Jaeger's storage needs.
 
-.Default Jaeger parameters for Elasticsearch in production
+Elasticsearch is a memory intensive application. The initial set of nodes specified in the default {product-title} installation may not be large enough to support the Elasticsearch cluster.  You should modify the default Elasticsearch configuration to match your use case and the resources you have requested for your {product-title} installation. You can adjust both the CPU and memory limits for each component by modifying the resources block with valid CPU and memory values. Additional nodes must be added to the  cluster if you want to run with the recommended amount (or more) of memory. Ensure that you do not exceed the resources requested for your {product-title} installation.
```

Comment:
s/memory intensive/memory-intensive ? 
I use _memory-intensive_ in the logging docs.